### DISCUSSION TITLE
Disabling long text wrapping to the right in side-by-side mode

### DISCUSSION THREAD

**Discussion by ghost** [dandavison/delta#1461] [Q&A]
Hello there, I was wondering if there's an inbuilt way to disable the wrapping symbols and text wrapping to the right in `side-by-side` mode. 
I was able to achieve this with a solution that is very much hacked together so a cleaner solution would be appreciated.
My solution:
```
[delta]
	wrap-max-lines=unlimited
	wrap-right-percent=1
	wrap-left-symbol=" "
```
I don't like this because of the extra character that has to be sacrificed. 

For reference, this is what it looks like by default:
<img width="459" alt="image" src="https://github.com/dandavison/delta/assets/85601323/136e604f-b26e-4377-b64d-df3978fa0867">

and this is what I want it to look like:
<img width="458" alt="image" src="https://github.com/dandavison/delta/assets/85601323/09d7fe8b-30b8-4fd0-9709-208d5ab88c98">

**Comment by th1000s** [comment]
Currently there is not, sorry, but I'll keep this in mind when I work on the wrapping logic next time.
